migrations/schema.sql

```diff
@@ -16,6 +16,42 @@ SET default_tablespace = '';
 SET default_with_oids = false;
 
 --
+-- Name: active_admin_comments; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+--
+
+CREATE TABLE active_admin_comments (
```

Comment:
If I drop the whole db, load a new one from production, then I should
npm run migrate
to run my migrations and then
 ./migrations/dump_schema.sh > ./migrations/schema.sql
and add the new schema to this pull request
?